content-services/content-sources-backend
# Fixes 2245: add red hat repo for nightly snapshotting

## Summary

Adds an intial set of Red hat repositories that is loaded from a json file.  
Adds a single gpg key loaded for all red hat repos
These repos get created into the repository_configs table with an org_id of -1

## Testing steps

load in ephemeral, or run `make repos-import`

1. curl  http://localhost:8000/api/content-sources/v1.0/repositories/  for some new org
  By default, nothing should be returned
2.  curl http://localhost:8000/api/content-sources/v1.0/repositories/?origin=red_hat
  you should get all the red hat repos
3.  create some repo
4. curl http://localhost:8000/api/content-sources/v1.0/repositories/?origin=red_hat,external
  you  should get both custom and redhat repos
